salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
And if you try 'utf-16' instead of 'utf-8'? Also that grains information?

If it's simpler drop into a python shell on the minion and do the following:

```python
❯ python                                                                                                                                                                     2015.5 f46a0da ✗
Python 2.7.8 (default, Oct 27 2014, 04:23:15) 
[GCC 4.9.1 20140903 (prerelease)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import salt
>>> print __salt_system_encoding__
UTF-8
>>> 
```